mathics/doc/common_doc.py

```diff
@@ -459,36 +462,25 @@ def __str__(self) -> str:
         return self.test
 
 
-class DocTests:
+# Tests has to appear before Documentation which uses it.
```

Comment:
I want this here because at some point I would like to DRY Tests and DocTests

Reply:
Please move it when you need it later then. This can be done after release.